lib/support/tcpdump.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { spawn } = require('child_process');
```

Comment:
We mostly try to use [execa](https://www.npmjs.com/package/execa) since it supports promises and have some better defaults than spawn.